### ISSUE TITLE
Use of Vendor namespaces

### ISSUE THREAD

**Issue by martinokeefe** [dmn-tck/tck#479]
Is the use of the vendor namespace required/ correct/ appropriate?

https://github.com/dmn-tck/tck/blob/89e71b1513e14006e70301f533b56662099af3dc/TestCases/compliance-level-3/1130-feel-interval/1130-feel-interval.dmn

See xmlns:triso="http://www.trisotech.com/2015/triso/modeling" in the namespace imports and subsequent usages.

**Comment by tarilabs** [comment]
The XML is valid so I don't see reasons why this is problematic.

It is not _required_, since we perform XSD validation (against the DMN specification XSD) and indeed TCK testing on purely DMN semantic.
This is an additional namespace prefix (`:triso`) which would pertain on the XML domain only to editor-relevant additional metadata attribute.

It is _correct_ , since this is allowed by the DMN spec and the XML standards, and proved by the XSD validation.

With regards to _appropriate_ this is a qualitative judgement; so I believe it would be better to specify why you might consider this not-appropriate, if that's the case?

In conclusion, the XSD validation against the DMN mandated XSD is passing, and the DMN model is checked against the semantic, without any impact from additional, editor-metadata xml attributes.

Hope this clarifies!
Let us know

**Comment by StrayAlien** [comment]
my $0.02?  I would remove them.  I think they add clutter but no add value in the TCK XMLs.  My feeling is the tests should have exactly only exactly what is needed for TCK duties, not remnants of specific tooling.  

I am sure it is not the case, but consider, if a vendor has meta data in the model, do _they_ require it in order to pass the tests?  

Not religious about it, and they do not hurt anything.  Just IMO.

**Comment by falko** [comment]
The testing of extension handling is very important. Thus I vote for keeping them. It also keeps models real, i.e. as in generated by an actual tool and not handcrafted in an XML editor.

**Comment by tarilabs** [comment]
DMN TCK 2021-12-14

discussed with the group and agreed not-an-issue per above discussions.

Also to be reminded the TCK currently does not test the Diagram Interchange.
From engine pov is good to check that implementations do not break with extensions provided by different modelers/editors and in fact execute as inteded (per above [comment](https://github.com/dmn-tck/tck/issues/479#issuecomment-963622637) by Falko)

**Comment by tarilabs** [comment]
I should have also noted, even the DMN examples from the DMN specification itself do contain vendor extension examples.
